cmd/jujud/agent/controllercharm.go

```diff
@@ -342,5 +408,16 @@ func addControllerApplication(
 	if err != nil {
 		return nil, errors.Trace(err)
 	}
+
+	// Expose all endpoints of the controller app
```

Comment:
Should we be exposing all endpoints of the controller app?

Reply:
We don't have an ssh endpoint so we have no control via spaces over what CIDRs to use so for now we can start with `0.0.0.0` but should consider tightening this up as part of the work.